Sources/LanguageServerProtocol/Notifications/WorkDoneProgress.swift

```diff
@@ -10,9 +10,22 @@
 //
 //===----------------------------------------------------------------------===//
 
-public enum WorkDoneProgress: NotificationType, Hashable {
+public struct WorkDoneProgress: NotificationType, Hashable {
   public static var method: String = "$/progress"
 
+  /// The progress token provided by the client or server.
+  public var token: ProgressToken
+
+  /// The progress data.
+  public var value: WorkDoneProgressType
+
+  public init(token: ProgressToken, value: WorkDoneProgressType) {
```

Comment:
Is this initializer used anywhere outside of the tests? I don't see any other references to it in this PR.

Reply:
No due to #577. I use your type definitions in another language server and want  to use $/progress => I implement it and send it upstream

Reply:
No due to #577. I use your type definitions in another language server and want  to use $/progress => I implement it and send it upstream